mlir/lib/Conversion/AMDGPUToROCDL/AMDGPUToROCDL.cpp

```diff
@@ -2730,6 +2731,18 @@ struct ConvertAMDGPUToROCDLPass
     });
 
     populateAMDGPUToROCDLConversionPatterns(converter, patterns, *maybeChipset);
+    populateGpuMemorySpaceAttributeConversions(
```

Comment:
You'll probably also want to call the conversion function defined below so we can get buffer_fat_pointe, and you might want to see if there's common infrastructure here with the ../GPUToROCDL/ flow.

(... side thing I just realized - the conversion of the tensor descripor should be in that type conversion populator below so that said GPUToROCDL flow picks it up)

Reply:
`populateAMDGPUMemorySpaceAttributeConversions` is already called in`populateAMDGPUToROCDLConversionPatterns`, I can move `populateGpuMemorySpaceAttributeConversions` there too.